docs/xpack.rst

```diff
@@ -202,15 +202,3 @@ saves it to a destination index.
 .. autoclass:: TransformClient
    :members:
 
-
```

Comment:
Let's handle this outside this PR so I can more easily backport to 7.x

Reply:
Done